src/main/kotlin/io/gitlab/arturbosch/detekt/idea/config/DetektConfig.kt

```diff
@@ -1,28 +1,35 @@
 package io.gitlab.arturbosch.detekt.idea.config
 
-import com.intellij.codeInsight.daemon.DaemonCodeAnalyzer
-import com.intellij.openapi.options.SearchableConfigurable
+import com.intellij.openapi.components.service
+import com.intellij.openapi.options.BoundSearchableConfigurable
 import com.intellij.openapi.project.Project
-import io.gitlab.arturbosch.detekt.idea.DETEKT
-import javax.swing.JComponent
-
-class DetektConfig(private val project: Project) : SearchableConfigurable {
-
-    private val detektConfigStorage: DetektConfigStorage = DetektConfigStorage.instance(project)
-    private val detektConfigurationForm: DetektConfigurationForm = DetektConfigurationForm(project)
-
-    override fun isModified(): Boolean = !detektConfigurationForm.isNotModified
-
-    override fun getId(): String = "io.github.detekt.config"
-
-    override fun getDisplayName(): String = DETEKT
-
-    override fun apply() {
-        detektConfigurationForm.apply()
-        DaemonCodeAnalyzer.getInstance(project).restart()
```

Comment:
This line is important to reanalyze when changing any detekt settings.
You can open a file with an issue and change the `warningsAsErrors` setting and notice that the file is not analyzed.

Reply:
Ah I forgot to put that back in! Will do asap